internal/generate/clusterserviceversion/clusterserviceversion_updaters.go

```diff
@@ -212,7 +212,6 @@ func mutatingToWebhookDescription(webhook admissionregv1.MutatingWebhook) operat
 func applyCustomResources(c *collector.Manifests, csv *operatorsv1alpha1.ClusterServiceVersion) error {
 	examples := []json.RawMessage{}
 	for _, cr := range c.CustomResources {
-		fmt.Println("applying", cr.GetName())
```

Comment:
Why the printed info was removed?

Reply:
I forgot to remove these from a previous PR.